locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -1222,8 +1224,16 @@ protected void customizeTemplate(ConfigBag setup, ComputeService computeService,
                     tags.add(AbstractCloudMachineNamer.sanitize(entry.getKey())+":"+AbstractCloudMachineNamer.sanitize(entry.getValue()));
                 }
                 slT.tags(tags);
+
+                if (!md.containsKey("notes")) {
+                    String notes = "User Metadata\n=============\n\n  * " + Joiner.on("\n  * ").withKeyValueSeparator(": ").join(md);
+                    if (notes.length() > NOTES_MAX_LENGTH) {
+                        String truncatedMsg = "...\n<truncated>";
```

Comment:
Agree, but then the truncate point depends on the length of the message which depends on the truncate point :). Not worth the effort of getting it right. I can easily add the total length so don't mind doing it.